website/highlights/2020-05-05-add-support-for-include-exclude-units.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I don't think this highlight is accurate, we [currently offer a `units` option](https://vector.dev/docs/reference/sources/journald/#units) on the `journald` source. What changed appears to be the fact that we split them to `include_units` and `exclude_units`.

Reply:
Wow, nice job, Ana.